include/spirv/unified1/spirv.core.grammar.json

```diff
@@ -10707,6 +10707,15 @@
           "capabilities" : [ "FPGAKernelAttributesINTEL" ],
           "version" : "None"
         },
+        {
+          "enumerant" : "StreamingInterfaceINTEL",
+          "value" : 6154,
+          "parameters" : [
+            { "kind" : "LiteralInteger", "name" : "'streaming_interface'" }
```

Comment:
Very minor: The name in the extension spec is "StallFreeReturn".  Should the name in the grammar file match?  If so, should the grammar file be updated (this PR) or should the extension spec be updated?

FWIW the name "streaming_interface" looks more consistent with the other names in the extension.